src/routes/(app)/+page.server.ts

```diff
@@ -40,3 +41,21 @@ export const load = async ({ locals }) => {
 		layoutReset: cmsPage.fullScreen
 	};
 };
+
+export const actions = {
+	navigate: async ({ locals }) => {
+		await collections.sessions.findOneAndUpdate(
+			{
+				sessionId: locals.sessionId
+			},
+			{
+				$set: {
+					updatedAt: new Date(),
+					acceptAgeLimitation: true
+				}
+			},
+			{ upsert: true }
```

Comment:
Since you're upserting, you should add all fields with `$setOnInsert` so that a newly inserted session has all required fields.

(For example, `createdAt`)